My commit is c9ca78b6 Add run_overhead_microbenchmark.sh (/root/dev/models/deploy)

==========================================================================
My environment variables:
--------------------------------------------------------------------------
CUDNN_VERSION=7.6.2.24-1+cuda10.1
PMIX_ID=1281097729.0
BERT_CONFIG_FILE=/root/dev/dataset/bert/uncased_L-12_H-768_A-12/bert_config.json
LD_LIBRARY_PATH=/usr/local/lib:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/nvidia/lib:/usr/local/nvidia/lib64
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
USE_RANK_FOR_CVD=true
OMPI_FIRST_RANKS=0
MPI_HOSTS=113f839a85ff,dafd9a5d0ab9
OMPI_MCA_orte_top_session_dir=/tmp/ompi.dafd9a5d0ab9.0
LOG_MEMORY_ENABLED=false
PMIX_SYSTEM_TMPDIR=/tmp
EVAL_BATCH_SIZE=8
OMPI_MCA_orte_num_nodes=1
OMPI_COMMAND=run_bert_glue.sh
MODEL=bert
LANG=C.UTF-8
HFI_NO_BACKTRACE=1
HOSTNAME=dafd9a5d0ab9
OMPI_MCA_ess_base_vpid=0
OLDPWD=/root/dev/logs
OMPI_MCA_initial_wdir=/root/dev/models/deploy
ENVIRONMENT=docker
TF_DIR=/root/dev/tensorflow
USE_KERAS_COMPILE_FIT=true
DATA_DIR=/root/dev/dataset/bert/finetuning
HOST_FLAG=--host 113f839a85ff,dafd9a5d0ab9
OMPI_MCA_btl_tcp_if_exclude=lo,docker0,eth1
DTYPE=fp32
NUM_CHECKPOINTS_TO_KEEP=5
OMPI_COMM_WORLD_NODE_RANK=0
NVIDIA_VISIBLE_DEVICES=all
OMPI_MCA_orte_precondition_transports=ec43f29bc304a16b-aa11b98c34fcd88f
BASE_DIR=/root/dev
OMPI_MCA_orte_ess_node_rank=0
OMPI_MCA_shmem_RUNTIME_QUERY_hint=mmap
OMPI_COMM_WORLD_RANK=0
LOG_STEPS=1
PMIX_RANK=0
OMPI_FILE_LOCATION=/tmp/ompi.dafd9a5d0ab9.0/pid.15840/0/0
TRAIN_BATCH_SIZE=8
NCCL_VERSION=2.4.7-1+cuda10.1
TF_FORCE_GPU_ALLOW_GROWTH=true
OMPI_MCA_orte_local_daemon_uri=1281097728.0;tcp://10.0.1.2,172.19.0.3:42403
GLUE_TASK=CoLA
LEARNING_RATE=2e-5
HOROVOD_GPU_OPERATIONS=NCCL
PWD=/root/dev/models/deploy
NUM_GPUS_PER_NODE=8
HOME=/root
ENABLE_XLA=true
PMIX_SERVER_TMPDIR=/tmp/ompi.dafd9a5d0ab9.0/pid.15840
RUN_SCRIPT=run_bert_glue.sh
PMIX_PTL_MODULE=tcp,usock
IPATH_NO_BACKTRACE=1
JOB_NAME=bert-glue-8bs-false-05_01_21_1619911213467
OMPI_MCA_rmaps_base_oversubscribe=1
OMPI_MCA_orte_launch=1
CODE_DIR=/root/dev/models/official/nlp/bert
OMPI_MCA_orte_tmpdir_base=/tmp
LOG_DIR=/root/dev/logs
TIMESTAMP=05_01_21_1619911214749
MPI_HOST_FILE=/root/container_hosts/hosts.txt
NUM_VIRTUAL_NODES_PER_DEVICE=1
BERT_MODEL_NAME=uncased_L-12_H-768_A-12
OMPI_APP_CTX_NUM_PROCS=1
OMPI_MCA_orte_output_filename=/root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467
LOG_FILE=/dev/stderr
STEPS_PER_LOOP=1
MODELS_DIR=/root/dev/models
OMPI_MCA_orte_app_num=0
DISABLE_VIRTUAL_NODES=false
PMIX_MCA_mca_base_component_show_load_errors=1
NUM_NODES=1
OMPI_MCA_pml=ob1
PYTHON_COMMAND=/usr/bin/python3
IN_DOCKER_CONTAINER=true
BERT_MODEL_DIR=/root/dev/dataset/bert/uncased_L-12_H-768_A-12
NUM_EPOCHS=1
PMIX_BFROP_BUFFER_TYPE=PMIX_BFROP_BUFFER_NON_DESC
MPI_HOME=/usr/local
CHECKPOINT_NAME=bert_model.ckpt
TERM=xterm
CUDA_PKG_VERSION=10-1=10.1.243-1
CUDA_VERSION=10.1.243
SKIP_EVAL=true
PMIX_DSTORE_ESH_BASE_PATH=/tmp/ompi.dafd9a5d0ab9.0/pid.15840/pmix_dstor_ds12_15840
OMPI_MCA_hwloc_base_binding_policy=none
OMPI_MCA_rmaps_base_mapping_policy=node
PMIX_SERVER_URI3=1281097728.0;tcp4://127.0.0.1:40273
PMIX_SERVER_URI2=1281097728.0;tcp4://127.0.0.1:40273
NUM_STEPS=20
OMPI_MCA_orte_hnp_uri=1281097728.0;tcp://10.0.1.2,172.19.0.3:42403
TRAIN_DIR=/root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467
OMPI_MCA_mpi_yield_when_idle=0
NVIDIA_DRIVER_CAPABILITIES=compute,utility
DEFAULT_NUM_GPUS_PER_NODE=8
ENABLE_CHECKPOINTS=false
DISTRIBUTION_STRATEGY=mirrored
OMPI_COMM_WORLD_LOCAL_SIZE=1
CUDA_VISIBLE_DEVICES=0
OMPI_COMM_WORLD_SIZE=1
SHLVL=4
PYTHONPATH=$PYTHONPATH:/root/dev/models:/root/dev/models:/root/dev/models
OMPI_NUM_APP_CTX=1
NVIDIA_REQUIRE_CUDA=cuda>=10.1 brand=tesla,driver>=396,driver<397 brand=tesla,driver>=410,driver<411 brand=tesla,driver>=418,driver<419
OMPI_MCA_pmix=^s1,s2,cray,isolated
DEFAULT_DISTRIBUTION_STRATEGY=mirrored
RUN_TAG=8bs-false
PMIX_SERVER_URI21=1281097728.0;tcp4://127.0.0.1:40273
SAVED_CHECKPOINT_PATH=/root/dev/dataset/bert/uncased_L-12_H-768_A-12/bert_model.ckpt
PMIX_DSTORE_21_BASE_PATH=/tmp/ompi.dafd9a5d0ab9.0/pid.15840/pmix_dstor_ds21_15840
ENABLE_MONITOR_MEMORY=false
CUBLAS_VERSION=10.1.0.105-1
BATCH_SIZE=8
PATH=/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PMIX_SECURITY_MODE=native
OMPI_MCA_ess=^singleton
NCCL_DEBUG=INFO
PMIX_NAMESPACE=1281097729
OMPI_MCA_orte_ess_num_procs=1
NCCL_SOCKET_IFNAME=^lo,docker0,eth1
OMPI_MCA_ess_base_jobid=1281097729
OMPI_COMM_WORLD_LOCAL_RANK=0
OMPI_UNIVERSE_SIZE=2
OMPI_MCA_orte_jobfam_session_dir=/tmp/ompi.dafd9a5d0ab9.0/pid.15840
DEFAULT_NUM_EPOCHS=1
PMIX_GDS_MODULE=ds21,ds12,hash
ENABLE_ELASTICITY=false
NUM_GPUS=1
OMPI_MCA_btl=^openib
_=/usr/bin/printenv
==========================================================================

2021-05-01 23:20:15.081987: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
/usr/local/lib/python3.6/dist-packages/tensorflow_addons/utils/ensure_tf_install.py:68: UserWarning: Tensorflow Addons supports using Python ops for all Tensorflow versions above or equal to 2.2.0 and strictly below 2.4.0 (nightly versions are not supported). 
 The versions of TensorFlow you are currently using is 2.4.0-rc3 and is not supported. 
Some things might work, some things might not.
If you were to encounter a bug, do not file an issue.
If you want to make sure you're using a tested and supported configuration, either change the TensorFlow version or the TensorFlow Addons's version. 
You can find the compatibility matrix in TensorFlow Addon's readme:
https://github.com/tensorflow/addons
  UserWarning,
/usr/local/lib/python3.6/dist-packages/absl/flags/_validators.py:359: UserWarning: Flag --model_dir has a non-None default value; therefore, mark_flag_as_required will pass even if flag is not specified in the command line!
  'command line!' % flag_name)
I0501 23:20:17.652210 140384857352000 virtual_helper.py:221] Setting TF_CONFIG to {"cluster": {"worker": ["dafd9a5d0ab9:2222"]}, "task": {"type": "worker", "index": 0}}
INFO:tensorflow:Initializing local devices since in-graph multi-worker training with `MirroredStrategy` is not supported in eager mode. TF_CONFIG will be ignored when when initializing `MirroredStrategy`.
I0501 23:20:17.653056 140384857352000 mirrored_strategy.py:306] Initializing local devices since in-graph multi-worker training with `MirroredStrategy` is not supported in eager mode. TF_CONFIG will be ignored when when initializing `MirroredStrategy`.
2021-05-01 23:20:17.653332: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-05-01 23:20:17.653443: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcuda.so.1
2021-05-01 23:20:24.971446: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:03:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.665GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-05-01 23:20:24.971521: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-01 23:20:24.975621: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
2021-05-01 23:20:24.975720: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.10
2021-05-01 23:20:24.978490: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-05-01 23:20:24.978920: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-05-01 23:20:24.981752: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-05-01 23:20:24.983258: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.10
2021-05-01 23:20:24.989059: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.7
2021-05-01 23:20:24.990996: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-05-01 23:20:24.993798: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-05-01 23:20:24.994803: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:03:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.665GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-05-01 23:20:24.994829: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-01 23:20:24.994857: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
2021-05-01 23:20:24.994880: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.10
2021-05-01 23:20:24.994901: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-05-01 23:20:24.994922: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-05-01 23:20:24.994942: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-05-01 23:20:24.994963: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.10
2021-05-01 23:20:24.994984: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.7
2021-05-01 23:20:24.996790: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-05-01 23:20:24.996836: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-05-01 23:20:25.869996: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1261] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-05-01 23:20:25.870077: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1267]      0 
2021-05-01 23:20:25.870088: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1280] 0:   N 
2021-05-01 23:20:25.872330: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:39] Overriding allow_growth setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
2021-05-01 23:20:25.872394: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10097 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:03:00.0, compute capability: 7.5)
INFO:tensorflow:Using MirroredStrategy with devices ('/job:localhost/replica:0/task:0/device:GPU:0',)
I0501 23:20:25.886086 140384857352000 mirrored_strategy.py:350] Using MirroredStrategy with devices ('/job:localhost/replica:0/task:0/device:GPU:0',)
2021-05-01 23:20:25.891901: I tensorflow/core/profiler/lib/profiler_session.cc:136] Profiler session initializing.
2021-05-01 23:20:25.891927: I tensorflow/core/profiler/lib/profiler_session.cc:155] Profiler session started.
2021-05-01 23:20:25.891975: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1365] Profiler found 1 GPUs
2021-05-01 23:20:25.893389: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcupti.so.10.1
2021-05-01 23:20:25.993906: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1415] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI_ERROR_INSUFFICIENT_PRIVILEGES
2021-05-01 23:20:25.993978: I tensorflow/core/profiler/lib/profiler_session.cc:172] Profiler session tear down.
I0501 23:20:25.994157 140384857352000 run_classifier.py:187] Training using TF 2.x Keras compile/fit API with distribution strategy.
I0501 23:20:29.293079 140384857352000 optimization.py:89] using Adamw optimizer
I0501 23:20:29.293468 140384857352000 optimization.py:142] gradient_clip_norm=1.000000
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.401808 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.402956 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.406998 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.407769 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
WARNING:tensorflow:The argument `steps_per_execution` is no longer experimental. Pass `steps_per_execution` instead of `experimental_steps_per_execution`.
W0501 23:20:30.408624 140384857352000 training.py:532] The argument `steps_per_execution` is no longer experimental. Pass `steps_per_execution` instead of `experimental_steps_per_execution`.
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.424270 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.425060 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.426514 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:30.427284 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
2021-05-01 23:20:30.465108: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)
2021-05-01 23:20:30.532938: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 2095335000 Hz
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:36.326144 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
INFO:tensorflow:Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
I0501 23:20:36.328331 140384857352000 cross_device_ops.py:565] Reduce to /job:localhost/replica:0/task:0/device:CPU:0 then broadcast to ('/job:localhost/replica:0/task:0/device:CPU:0',).
2021-05-01 23:20:49.672793: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7fab50003fe0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2021-05-01 23:20:49.672881: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce RTX 2080 Ti, Compute Capability 7.5
2021-05-01 23:20:49.991528: I tensorflow/compiler/jit/xla_compilation_cache.cc:333] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
Training on virtual node 1 / 1
2021-05-01 23:20:50.709141: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
 1/20 [>.............................] - ETA: 13:52 - loss: 0.6715 - accuracy: 0.5000I0501 23:21:14.389773 140384857352000 keras_utils.py:163] TimeHistory: 43.8026 seconds, 0.1826 examples/second between steps 0 and 1
2021-05-01 23:21:15.052696: I tensorflow/core/profiler/lib/profiler_session.cc:136] Profiler session initializing.
2021-05-01 23:21:15.052747: I tensorflow/core/profiler/lib/profiler_session.cc:155] Profiler session started.
2021-05-01 23:21:15.054601: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1415] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI_ERROR_INSUFFICIENT_PRIVILEGES
Training on virtual node 1 / 1
 2/20 [==>...........................] - ETA: 13s - loss: 0.6607 - accuracy: 0.6250  I0501 23:21:15.154741 140384857352000 keras_utils.py:163] TimeHistory: 0.1022 seconds, 78.2824 examples/second between steps 1 and 2
2021-05-01 23:21:15.159222: I tensorflow/core/profiler/lib/profiler_session.cc:71] Profiler session collecting data.
2021-05-01 23:21:15.171850: I tensorflow/core/profiler/internal/gpu/cupti_collector.cc:228]  GpuTracer has collected 0 callback api events and 0 activity events. 
2021-05-01 23:21:15.175645: I tensorflow/core/profiler/lib/profiler_session.cc:172] Profiler session tear down.
2021-05-01 23:21:15.180262: I tensorflow/core/profiler/rpc/client/save_profile.cc:137] Creating directory: /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15
2021-05-01 23:21:15.183024: I tensorflow/core/profiler/rpc/client/save_profile.cc:143] Dumped gzipped tool data for trace.json.gz to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.trace.json.gz
2021-05-01 23:21:15.218315: I tensorflow/core/profiler/rpc/client/save_profile.cc:137] Creating directory: /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15
2021-05-01 23:21:15.225243: I tensorflow/core/profiler/rpc/client/save_profile.cc:143] Dumped gzipped tool data for memory_profile.json.gz to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.memory_profile.json.gz
2021-05-01 23:21:15.225631: I tensorflow/core/profiler/rpc/client/capture_profile.cc:251] Creating directory: /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15Dumped tool data for xplane.pb to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.xplane.pb
Dumped tool data for overview_page.pb to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.overview_page.pb
Dumped tool data for input_pipeline.pb to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.input_pipeline.pb
Dumped tool data for tensorflow_stats.pb to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.tensorflow_stats.pb
Dumped tool data for kernel_stats.pb to /root/dev/logs/bert-glue-8bs-false-05_01_21_1619911213467/summaries/train/plugins/profile/2021_05_01_23_21_15/dafd9a5d0ab9.kernel_stats.pb

Training on virtual node 1 / 1
 3/20 [===>..........................] - ETA: 7s - loss: 0.6575 - accuracy: 0.6667 I0501 23:21:15.319827 140384857352000 keras_utils.py:163] TimeHistory: 0.0928 seconds, 86.2358 examples/second between steps 2 and 3
Training on virtual node 1 / 1
 4/20 [=====>........................] - ETA: 5s - loss: 0.6568 - accuracy: 0.6797I0501 23:21:15.401108 140384857352000 keras_utils.py:163] TimeHistory: 0.0789 seconds, 101.3389 examples/second between steps 3 and 4
Training on virtual node 1 / 1
 5/20 [======>.......................] - ETA: 4s - loss: 0.6561 - accuracy: 0.6888I0501 23:21:15.481780 140384857352000 keras_utils.py:163] TimeHistory: 0.0781 seconds, 102.4381 examples/second between steps 4 and 5
Training on virtual node 1 / 1
 6/20 [========>.....................] - ETA: 3s - loss: 0.6577 - accuracy: 0.6816I0501 23:21:15.561687 140384857352000 keras_utils.py:163] TimeHistory: 0.0778 seconds, 102.8510 examples/second between steps 5 and 6
WARNING:tensorflow:Callback method `on_train_batch_end` is slow compared to the batch time (batch time: 0.0848s vs `on_train_batch_end` time: 0.1250s). Check your callbacks.
W0501 23:21:15.564198 140384857352000 callbacks.py:340] Callback method `on_train_batch_end` is slow compared to the batch time (batch time: 0.0848s vs `on_train_batch_end` time: 0.1250s). Check your callbacks.
Training on virtual node 1 / 1
 7/20 [=========>....................] - ETA: 2s - loss: 0.6593 - accuracy: 0.6761I0501 23:21:15.643466 140384857352000 keras_utils.py:163] TimeHistory: 0.0789 seconds, 101.4566 examples/second between steps 6 and 7
Training on virtual node 1 / 1
 8/20 [===========>..................] - ETA: 2s - loss: 0.6609 - accuracy: 0.6697I0501 23:21:15.720949 140384857352000 keras_utils.py:163] TimeHistory: 0.0751 seconds, 106.5474 examples/second between steps 7 and 8
Training on virtual node 1 / 1
 9/20 [============>.................] - ETA: 1s - loss: 0.6616 - accuracy: 0.6663I0501 23:21:15.800424 140384857352000 keras_utils.py:163] TimeHistory: 0.0765 seconds, 104.6336 examples/second between steps 8 and 9
Training on virtual node 1 / 1
10/20 [==============>...............] - ETA: 1s - loss: 0.6624 - accuracy: 0.6621I0501 23:21:15.878859 140384857352000 keras_utils.py:163] TimeHistory: 0.0756 seconds, 105.7845 examples/second between steps 9 and 10
Training on virtual node 1 / 1
11/20 [===============>..............] - ETA: 1s - loss: 0.6634 - accuracy: 0.6577I0501 23:21:15.958336 140384857352000 keras_utils.py:163] TimeHistory: 0.0764 seconds, 104.7372 examples/second between steps 10 and 11
Training on virtual node 1 / 1
12/20 [=================>............] - ETA: 1s - loss: 0.6640 - accuracy: 0.6550I0501 23:21:16.037735 140384857352000 keras_utils.py:163] TimeHistory: 0.0769 seconds, 104.0912 examples/second between steps 11 and 12
Training on virtual node 1 / 1
13/20 [==================>...........] - ETA: 1s - loss: 0.6643 - accuracy: 0.6542I0501 23:21:16.117762 140384857352000 keras_utils.py:163] TimeHistory: 0.0769 seconds, 104.0832 examples/second between steps 12 and 13
Training on virtual node 1 / 1
14/20 [====================>.........] - ETA: 0s - loss: 0.6650 - accuracy: 0.6521I0501 23:21:16.196539 140384857352000 keras_utils.py:163] TimeHistory: 0.0756 seconds, 105.7509 examples/second between steps 13 and 14
Training on virtual node 1 / 1
15/20 [=====================>........] - ETA: 0s - loss: 0.6657 - accuracy: 0.6492I0501 23:21:16.277566 140384857352000 keras_utils.py:163] TimeHistory: 0.0784 seconds, 101.9761 examples/second between steps 14 and 15
Training on virtual node 1 / 1
16/20 [=======================>......] - ETA: 0s - loss: 0.6662 - accuracy: 0.6472I0501 23:21:16.358028 140384857352000 keras_utils.py:163] TimeHistory: 0.0773 seconds, 103.4708 examples/second between steps 15 and 16
Training on virtual node 1 / 1
17/20 [========================>.....] - ETA: 0s - loss: 0.6666 - accuracy: 0.6459I0501 23:21:16.437639 140384857352000 keras_utils.py:163] TimeHistory: 0.0772 seconds, 103.6594 examples/second between steps 16 and 17
Training on virtual node 1 / 1
18/20 [==========================>...] - ETA: 0s - loss: 0.6670 - accuracy: 0.6447I0501 23:21:16.516926 140384857352000 keras_utils.py:163] TimeHistory: 0.0765 seconds, 104.6101 examples/second between steps 17 and 18
Training on virtual node 1 / 1
19/20 [===========================>..] - ETA: 0s - loss: 0.6673 - accuracy: 0.6437I0501 23:21:16.596820 140384857352000 keras_utils.py:163] TimeHistory: 0.0768 seconds, 104.1905 examples/second between steps 18 and 19
Training on virtual node 1 / 1
20/20 [==============================] - ETA: 0s - loss: 0.6675 - accuracy: 0.6430I0501 23:21:16.673770 140384857352000 keras_utils.py:163] TimeHistory: 0.0745 seconds, 107.4388 examples/second between steps 19 and 20
20/20 [==============================] - 46s 120ms/step - loss: 0.6676 - accuracy: 0.6425
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-4._attention_layer._masked_softmax
W0501 23:21:19.559566 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-4._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-4._attention_layer._dropout
W0501 23:21:19.559766 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-4._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-5._attention_layer._masked_softmax
W0501 23:21:19.559842 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-5._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-5._attention_layer._dropout
W0501 23:21:19.559913 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-5._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-6._attention_layer._masked_softmax
W0501 23:21:19.559982 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-6._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-6._attention_layer._dropout
W0501 23:21:19.560048 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-6._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-7._attention_layer._masked_softmax
W0501 23:21:19.560114 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-7._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-7._attention_layer._dropout
W0501 23:21:19.560180 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-7._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-8._attention_layer._masked_softmax
W0501 23:21:19.560246 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-8._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-8._attention_layer._dropout
W0501 23:21:19.560311 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-8._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-9._attention_layer._masked_softmax
W0501 23:21:19.560377 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-9._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-9._attention_layer._dropout
W0501 23:21:19.560441 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-9._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-10._attention_layer._masked_softmax
W0501 23:21:19.560508 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-10._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-10._attention_layer._dropout
W0501 23:21:19.560582 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-10._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-11._attention_layer._masked_softmax
W0501 23:21:19.560653 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-11._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-11._attention_layer._dropout
W0501 23:21:19.560720 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-11._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-12._attention_layer._masked_softmax
W0501 23:21:19.560789 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-12._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-12._attention_layer._dropout
W0501 23:21:19.560855 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-12._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-13._attention_layer._masked_softmax
W0501 23:21:19.560922 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-13._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-13._attention_layer._dropout
W0501 23:21:19.560986 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-13._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-14._attention_layer._masked_softmax
W0501 23:21:19.561053 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-14._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-14._attention_layer._dropout
W0501 23:21:19.561118 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-14._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-15._attention_layer._masked_softmax
W0501 23:21:19.561184 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-15._attention_layer._masked_softmax
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-15._attention_layer._dropout
W0501 23:21:19.561249 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-15._attention_layer._dropout
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-4._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561318 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-4._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-5._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561386 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-5._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-6._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561452 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-6._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-7._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561517 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-7._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-8._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561582 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-8._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-9._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561648 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-9._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-10._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561713 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-10._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-11._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561778 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-11._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-12._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561843 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-12._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-13._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561908 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-13._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-14._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.561974 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-14._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:Unresolved object in checkpoint: (root).model.layer_with_weights-15._attention_layer._masked_softmax._mask_expansion_axes
W0501 23:21:19.562039 140384857352000 util.py:161] Unresolved object in checkpoint: (root).model.layer_with_weights-15._attention_layer._masked_softmax._mask_expansion_axes
WARNING:tensorflow:A checkpoint was restored (e.g. tf.train.Checkpoint.restore or tf.keras.Model.load_weights) but not all checkpointed values were used. See above for specific issues. Use expect_partial() on the load status object, e.g. tf.train.Checkpoint.restore(...).expect_partial(), to silence these warnings, or use assert_consumed() to make the check explicit. See https://www.tensorflow.org/guide/checkpoint#loading_mechanics for details.
W0501 23:21:19.562107 140384857352000 util.py:169] A checkpoint was restored (e.g. tf.train.Checkpoint.restore or tf.keras.Model.load_weights) but not all checkpointed values were used. See above for specific issues. Use expect_partial() on the load status object, e.g. tf.train.Checkpoint.restore(...).expect_partial(), to silence these warnings, or use assert_consumed() to make the check explicit. See https://www.tensorflow.org/guide/checkpoint#loading_mechanics for details.
